README.md

```diff
@@ -39,18 +34,17 @@ Development on a solution within cm_config_tools is underway.
    need:
    
    ```bash
-   drush config-devel-module-dependencies MYMODULE --active=1 --direction=both
-   ```
-   
+   drush cm-config-tools-suggest MYMODULE
+      ```
+     
    This lists anything that is dependent on the config you have already listed
    (e.g. field instances for a node type), as well as the dependencies of that
    (e.g. field storage config, but also view mode settings). So copy the config
    you want to export into that `managed` section.
     
    Dependencies will get added where necessary when exporting.
    
-   Note that you may want to run that `drush config-devel-module-dependencies`
-   command again when done to check there are not more config names you should
+   Note that you may want to run that command again when done to check there are not more config names you should
```

Comment:
Please can we keep lines in this file wrapped at 80 chars where sane to do so? The same applies to a few other lines later in this file. Thanks!